### ISSUE TITLE
Could not determine active kernel version: ukuu does not start

### ISSUE THREAD

**Issue by timrichardson** [teejee2008/ukuu-plus#41]
I have a custom kernel, and it stops ukuu from doing anything useful.

```
... Ukuu Kernel Update Utility v19.12 by Tony George (teejeetech@gmail.com)

tim@moncrief:~$ uname -r
5.3.13-custom
tim@moncrief:~$ 

```
```

tim@moncrief:~$ ukuu --list-installed

(process:7946): GLib-CRITICAL **: 07:30:19.891: g_str_has_prefix: assertion 'prefix != NULL' failed

(process:7946): GLib-CRITICAL **: 07:30:19.891: g_str_has_prefix: assertion 'prefix != NULL' failed

(process:7946): GLib-CRITICAL **: 07:30:19.891: g_str_has_prefix: assertion 'prefix != NULL' failed

** (process:7946): CRITICAL **: 07:30:19.891: string_contains: assertion 'needle != NULL' failed

** (process:7946): CRITICAL **: 07:30:19.891: string_contains: assertion 'needle != NULL' failed

** (process:7946): CRITICAL **: 07:30:19.891: string_contains: assertion 'needle != NULL' failed

** (process:7946): CRITICAL **: 07:30:19.892: string_contains: assertion 'needle != NULL' failed
E: Could not determine active kernel version
E: Please report this issue
tim@moncrief:~$ 
```

**Comment by teejee2008** [comment]
You need to be running an official or mainline kernel to use Ukuu. If the kernel is custom it will not be able to determine the kernel version.

Send me the output of the following:
uname -r
uname -a

**Comment by philippgerard** [comment]
Same here with Xanmod kernels in use on Pop! OS.

```
user:~$ uname -r
5.5.10-xanmod2
user:~$ uname -a
Linux 5.5.10-xanmod2 #1 SMP PREEMPT Wed Mar 18 10:40:26 -03 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Would be helpful to list this kernel simply as "could not identify" or something, so you can still install/update other mainline kernels etc.

**Comment by teejee2008** [comment]
Fixed in 20.05
http://teejeetech.in/2020/05/24/ukuu-20-05/
